src/main/java/mil/dds/anet/database/ApprovalStepDao.java

```diff
@@ -153,6 +148,19 @@ public boolean deleteStep(String uuid) {
 		return true;
 	}
 
+	/**
+	 * Check whether the Approval Step is beign by a report
+	 */
+
+	public boolean isStepInUse(String uuid) {
```

Comment:
Method can be `private`.

Reply:
This method is also being used by OrganizationResource.isApprovalStepInUse, and thus public exposure is necessary.